### PR TITLE
Jenkins Cleanup

### PR DESCRIPTION
**Pull request by edgibbs**

### CODE REVIEW

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

File: Jenkinsfile [hunk 1]
```diff
@@ -95,129 +97,194 @@ def getNewTag(List tags, VersionIncrement increment) {
...
-		  git branch: 'master', credentialsId: GITHUB_CREDENTIALS_ID, url: GITHUB_URL
-	}
+def buildPullRequest() {
+  node('linux') {
+   try {
+     checkoutStage()
+     buildDockerImageStage()
+     checkForLabelPullRequest()
+     unitTestStage()
+     securityScanStage()
```
Comment (kelfink) [inline Jenkinsfile]: on every PR ?  Maybe not

File: Jenkinsfile [hunk 2]
```diff
@@ -95,129 +97,194 @@ def getNewTag(List tags, VersionIncrement increment) {
...
+      sh "docker exec -t ${container.id} sh -c 'AUTHENTICATION_API_BASE_URL=/api RAILS_ENV=test bundle exec rspec'"
+    }
+  }
+}
 
-                newTag = getNewTag(tags, increment)
-                debug("Increment Tag: newTag: ${newTag}")
+def securityScanStage() {
+  stage('Security Scan') {
+    containerScan('dashboard', '1.2.24')
```
Comment (kelfink) [inline Jenkinsfile]: surely a variable instead of 1.2.24

File: Jenkinsfile [hunk 3]
```diff
@@ -95,129 +97,194 @@ def getNewTag(List tags, VersionIncrement increment) {
...
 
-            }
-        }
-        catch(Exception ex) {
-            error "[ERROR] ${ex}"
-        }
+def incrementTagStage() {
+  stage("Increment Tag") {
+    def prEvent = getPullRequestEvent()
+    debug("Increment Tag: prEvent: ${prEvent}")
```
Comment (kelfink) [inline Jenkinsfile]: A bit noisy.  Maybe necessary to leave in debugs until this is all solid, down the road

Comment (edgibbs) [reply Jenkinsfile]: Planning on doing this when I move it into a shared library

**Pull request merged**